config/application.rb

```diff
@@ -8,19 +8,31 @@
 
 module Flashcards
   class Application < Rails::Application
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
+# Settings in config/environments/* take precedence over those specified here.
+# Application configuration should go into files in config/initializers
+# -- all .rb files in that directory are automatically loaded.
 
-    # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
-    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+# Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
+# Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
+# config.time_zone = 'Central Time (US & Canada)'
 
-    # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
-    # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
-    # config.i18n.default_locale = :de
+# The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
+# config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
+# config.i18n.default_locale = :de
+
+# Do not swallow errors in after_commit/after_rollback callbacks.
+
+  ActionView::Base.field_error_proc = Proc.new do |html_tag|
+    class_attr_index = html_tag.index 'class="'
+
+    if class_attr_index
+      html_tag.insert class_attr_index+7, 'error '
```

Comment:
Surrounding space missing for operator '+'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.